docs/scim/v1/scimv1.md

```diff
@@ -0,0 +1,361 @@
+
+| Endpoint                                | SCIM spec
+| :---:                                   | :---:         
+| {{baseUrl}}/scim/ServiceProviderConfig  | https://tools.ietf.org/html/rfc7643#section-5
+| {{baseUrl}}/scim/ResourceTypes          | https://tools.ietf.org/html/rfc7643#section-6
+| {{baseUrl}}/scim/Schemas                | https://tools.ietf.org/html/rfc7643#section-7
+
+Where **base_url** is `https://api-us.beyondidentity.com/v1/tenants/{{tenant_id}}/realms/{{realm_id}}/scim/v2`.
+
+##Operations
+The following operations are currently supported by BI SCIM server:
+
+- Create: POST {{baseUrl}}/{{resource}}
+- Read: GET {{baseUrl}}/{{resource}}/{{id}}
+- Replace: PUT {{baseUrl}}/{{resource}}/{{id}}
+- Delete: DELETE {{baseUrl}}/{{resource}}/{{id}}
+- Search: GET {{baseUrl}}/{{resource}}?ﬁlter={{attribute}}{{op}}{{value}}
+
+Where **base_url** is `https://api-us.beyondidentity.com/v1/tenants/{{tenant_id}}/realms/{{realm_id}}/scim/v2`. **resource** can either be `Groups`, `Users`. 
+**id** represents the BI's unique identifier of the corresponding resource.  
+
+###Users
+
+We support the following **Users**`(urn:ietf:params:scim:schemas:core:2.0:User)`attributes:
+
+| User Attribute | Attribute Type| Data Type|Required?|Mutability|Description
+| :---:          | :---:         | :---:    | :---:   | :---:    | :---:     
+| id             | SVA           | string   |N        |N         | ID is unique attribute that identifies the user within a realm.
+| userName       | SVA           | string   |Y        |Y         | The username of the user. The value of this field will be returned as the subject of a OIDC ID Token.
+| displayName    | SVA           | string   |Y        |Y         | The name of the User, suitable for display to end-users.  The name SHOULD be the full name of the User being described, if known.
+| emails         | CMVA          | n/a      |N        |Y         | Email for the User. Only one email is allowed. Primary must be set to true.
+| email.value    | SVA           | string   |Y        |Y         | Address of the user email.
+| email.primary  | SVA           | bool     |Y        |N         | Indicates if this email is primary email of the user.  Currently always set to `true`
+
+SVA = single value attribute
+CMVA = complex multi value attribute
+
+If unsupported attributes are specified in the request, they will be ignored.
+
+**Create user example**
+* Notes:
+    - If email is provided, `primary` must be set to `true` and `value` must be a valid email address
+
+Sample request
+```
+    POST /scim/v2/Users HTTP/1.1
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:ietf:params:scim:schemas:core:2.0:User"],
+        "userName": "test.user@bi.com",
+        "emails": [{
+            "primary": true,
+            "value": "test.user@bi.com",
+        }],
+        "displayName": "Test User"
+    }
+```
+Sample response
```

Comment:
Use heading syntax as much as possible. Leveraging this gives you an outline to the right side of the page. Your current outline looks like this:

<img width="252" alt="image" src="https://user-images.githubusercontent.com/6578679/200722585-b64130ad-2e76-4b67-94b6-6c6d64877057.png">

This is because only one heading in your document uses the heading syntax. Ideally, I should be able to jump to each example by just using the outline.